language-examples/java/rest/src/main/java/com/vectara/examples/rest/RestArgs.java

```diff
@@ -52,4 +52,9 @@ public class RestArgs {
       description = "API key retrieved from Vectara console")
   @Nullable
   String apiKey = null;
+  
+  @Parameter(
+      names= {"--query"},
+      description = "The query you want to send to the backend.")
```

Comment:
I think "the backend" here sounds a bit weird.  I think "to Vectara" sounds more natural

Reply:
fixed.